packages/kolibri/components/pages/AppBarPage/internal/AppBar.vue

```diff
@@ -222,9 +224,18 @@
       window.addEventListener('click', this.handleWindowClick);
       window.addEventListener('keydown', this.handlePopoverByKeyboard, true);
```

Comment:
Probably out of scope for this fix, but might be worth using the opportunity to move the event additions to the mounted hook instead?

Reply:
I am not against moving these event listeners to the mounted hook instead, I will update the PR with this change.